javascript-functions/main.js

```diff
@@ -0,0 +1,36 @@
+function convertMinutesToSeconds(minutes) {
+  var seconds = minutes * 60;
+  return seconds;
+}
+var secondsResult = convertMinutesToSeconds(6);
+console.log('Seconds:', secondsResult);
+
+function greet(name) {
+  var greeting = 'Hey, ' + name;
+  return greeting;
+}
+var greetingResult = greet('Jacob');
+console.log('Greeting:', greetingResult);
+
+function getArea(width, height) {
+  var area = width * height;
+  return area;
+}
+var areaResult = getArea(10, 10);
+console.log('Area:', areaResult);
+
+function getFirstName(person) {
+  person.firstName = 'Lelouche';
+  person.lastName = 'Lamperouge';
```

Comment:
Oh dear. Try calling this with an object that includes different names, you will see that this function will **_always_** return `"Lelouche"`, which is wrong.